package.json

```diff
@@ -109,7 +109,6 @@
     ],
     "*.css": [
       "stylelint",
-      "prettier --write",
```

Comment:
Had to remove this, because it was linting the css files in the pre-commit, causing them to fail on stylelint in the pre-push check.